<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>order</class>
 <widget class="QWidget" name="order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>800</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="allBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:checked
{
	background-color: rgb(85, 170, 255);
}
QPushButton{
background-color: rgb(216, 216, 216);
border: 1px solid rgb(255,255,255);
border-radius:10px;
}

</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="blueBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:checked
{
	background-color: rgb(85, 170, 255);
}
QPushButton{
background-color: rgb(216, 216, 216);
border: 1px solid rgb(255,255,255);
border-radius:10px;
}

</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="redBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:checked
{
	background-color: rgb(85, 170, 255);
}
QPushButton{
background-color: rgb(216, 216, 216);
border: 1px solid rgb(255,255,255);
border-radius:10px;
}

</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="yellowBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:checked
{
	background-color: rgb(85, 170, 255);
}
QPushButton{
background-color: rgb(216, 216, 216);
border: 1px solid rgb(255,255,255);
border-radius:10px;
}

</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="orderLogo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/Pic_/Picture/Logo_Signin.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>500</width>
     <height>70</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 155);
font: 12pt &quot;宋体&quot; ;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入菜品名称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchBtn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>0</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:10px solid rgb(255, 255, 255);
background-color: rgb(155, 155, 155);
font: 12pt &quot;宋体&quot; ;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>603</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 127);
border:1px solid rgb(255,255,255);
border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>check bill</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinNum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>603</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(180, 180, 180);
border-radius:5px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="verifyBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>603</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);
border:1px solid rgb(255,255,255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="orderBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>80</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);
border: 1px solid rgb(255,255,255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>609</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
